llvm/lib/SYCLLowerIR/CompileTimePropertiesPass.cpp

```diff
@@ -582,6 +589,11 @@ bool CompileTimePropertiesPass::transformSYCLPropertiesAnnotation(
   std::string NewAnnotString = "";
   auto Properties = parseSYCLPropertiesString(M, IntrInst);
   for (auto &Property : Properties) {
+    // sycl-alignment is converted to align in this
```

Comment:
Do we need to still call 'parseAlignmentAndApply'? Thanks

Reply:
yes we do. Comment updated